conda_forge_tick/cli.py

```diff
@@ -209,6 +237,62 @@ def make_import_to_package_mapping(
     import_to_pkg.main(ctx, max_artifacts)
 
 
+@main.command(name="parse-feedstock")
+@feedstock_option
+@existing_feedstock_metadata_option
+def parse_feedstock(
+    feedstock: str,
+    existing_feedstock_metadata: str,
+) -> None:
+    """
+    Parse a feedstock and update the feedstock metadata.
```

Comment:
This prints the new node attributes file and only sometimes (if the `feedstock` option is used - is that correct?) updates it on disk.

I would expect this to be explained here.